python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1484,6 +1486,24 @@ keys:
                       - str
                     min: 2
                     max: 64
+          evpn_ethernet_segment:
+            type: list
+            primary_key: domain
+            items:
+              type: dict
+              keys:
+                domain:
+                  type: str
+                  valid_values:
+                    - all
+                    - local
+                    - remote
+                identifier:
+                  type: str
+                  description: 10 octet ethernet segment identifier.
```

Comment:
we have this description for `identifier` for ethernet_interface and port-channels. Would it make sense to make it the same here?

```
 EVPN Ethernet Segment Identifier (Type 1 format).
```